tests/Operation/ListCollectionsFunctionalTest.php

```diff
@@ -34,6 +34,33 @@ public function testListCollectionsForNewlyCreatedDatabase()
         }
     }
 
+    public function testOffsetGet()
+    {
+        if (version_compare($this->getServerVersion(), '3.4.0', '<')) {
+            $this->markTestSkipped('idIndex and info are not supported');
+        }
+
+        $server = $this->getPrimaryServer();
+
+        $operation = new DropDatabase($this->getDatabaseName());
+        $operation->execute($server);
```

Comment:
Is this necessary? The base FunctionalTestCase class for Operations already drops the collection during setup.

Assuming this was copied from `testListCollectionsForNewlyCreatedDatabase()`, I think that was a unique case where we needed to ensure that the database was just created during the test.